source/cf/install-cf-plugins.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+if [ -n $TRAVIS ]
```

Comment:
The logic here is that if Travis is integrated, skip the installation of scaleover?